### PR TITLE
Mar 922

### PR DESCRIPTION
**Pull request by cad106uk**

### CODE REVIEW

File: api/user/models.py [hunk 1]
```diff
@@ -8,8 +8,7 @@
 from django.utils import timezone
 from django_filters import BaseInFilter
 
-from api.barriers.filters import BarrierFilterSet
-from api.barriers.models import Barrier
+from api.barriers import models as bar_models
```
Comment (jamespacileo) [inline api/user/models.py:11]: barrier_models

File: api/barriers/models.py [hunk 2]
```diff
@@ -1,21 +1,14 @@
...
-from api.barriers import validators
-from api.barriers.report_stages import REPORT_CONDITIONS, report_stage_status
-from api.barriers.utils import random_barrier_reference
+import django_filters
+from api.collaboration import models as col_models
 from api.commodities.models import Commodity
 from api.commodities.utils import format_commodity_code
 from api.core.exceptions import ArchivingException
 from api.core.models import BaseModel, FullyArchivableMixin
+from api.metadata import models as md_models
```
Comment (jamespacileo) [inline api/barriers/models.py:10]: metadata_models

File: api/barriers/models.py [hunk 3]
```diff
@@ -1,21 +1,14 @@
...
+from api.metadata import utils
```
Comment (jamespacileo) [inline api/barriers/models.py:11]: metadata_utils

File: api/barriers/models.py [hunk 4]
```diff
@@ -1,21 +1,14 @@
...
-from django.db.models import CASCADE, Q
-from django.utils import timezone
-from hashid_field import HashidAutoField
-from simple_history.models import HistoricalRecords
-
-from api.barriers import validators
-from api.barriers.report_stages import REPORT_CONDITIONS, report_stage_status
-from api.barriers.utils import random_barrier_reference
+import django_filters
+from api.collaboration import models as col_models
```
Comment (jamespacileo) [inline api/barriers/models.py:5]: collaboration_models

File: api/metadata/utils.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
 from mohawk import Sender
 from urlobject import URLObject
 
-from api.wto.models import WTOCommitteeGroup
+from api.wto import models
```
Comment (jamespacileo) [inline api/metadata/utils.py:10]: wto_models

**Review by jamespacileo — CHANGES_REQUESTED** [review]

File: api/metadata/utils.py (hunk 5 above)
Comment (cad106uk) [reply api/metadata/utils.py]: done.

File: api/barriers/models.py (hunk 4 above)
Comment (cad106uk) [reply api/barriers/models.py]: done.

File: api/barriers/models.py (hunk 3 above)
Comment (cad106uk) [reply api/barriers/models.py]: done

File: api/barriers/models.py (hunk 2 above)
Comment (cad106uk) [reply api/barriers/models.py]: done

File: api/user/models.py (hunk 1 above)
Comment (cad106uk) [reply api/user/models.py]: done.

**Review by jamespacileo — APPROVED** [review]

**Pull request merged**